cmd/root.go

```diff
@@ -95,7 +91,25 @@ func init() {
 	RootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "enable debug logging")
 	RootCmd.PersistentFlags().BoolVarP(&quiet, "quiet", "q", false, "disable progress updates")
 	RootCmd.PersistentFlags().BoolVar(&noColor, "no-color", false, "disable colored output")
+	RootCmd.PersistentFlags().StringVarP(&logFmt, "logformat", "f", "", "log output format")
```

Comment:
Could we make do without the shorthand `-f` here for now and start with only `--logformat`?

Reply:
Just pushed the change